src/DetectChanges/BCBreak/MethodBased/MethodBased.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\BackwardCompatibility\DetectChanges\BCBreak\MethodBased;
+
+use Roave\BackwardCompatibility\Changes;
+use Roave\BetterReflection\Reflection\ReflectionMethod;
+
+interface MethodBased
+{
+    public function __invoke(ReflectionMethod $fromMethod, ReflectionMethod $toMethod) : Changes;
```

Comment:
All checkers are now functional interfaces - didn't like the repeated name, nor the `compare()` wording, and settled with using just the wording from the class name